tests/suites/test_suite_lms.data

```diff
@@ -0,0 +1,274 @@
+
+LMS pyhsslms interop test #2
+# This test uses data from https://github.com/russhousley/pyhsslms due to the
+# limited amount of available test vectors for LMS. The private key is stored in
+# data_files/lms_pyhsslms_sha256_m32_h5_lmots_sha256_n32_w8_prv. Note that this signature
+# uses leaf key 1, so must be the second signature generated by the key if the
+# signature is to be reproduced. Message data is random. Note that hash-sigs
+# stores public keys and signatures in HSS form, which appends a 4-byte "levels"
+# word at the start of the key/sig. We strip these 4 bytes from the signature
+# and the public key before including them in a the test data.
+#
+# To reproduce the signature:
+# * sudo pip3 install pyhsslms
```

Comment:
Likewise, please don't give examples that contain `sudo pip3 ...`. `pip3 install --user pyhsslms`.